### PR TITLE
add release notes for flexvolume plugin

### PR DESCRIPTION
**Pull request by raunakkumar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,9 +13,9 @@ HPE Volume Driver for Kubernetes FlexVolume Plugin leverages HPE Nimble Storage
 
 ### HPE Nimble Storage
 
-| Driver                  | HPE Nimble Storage Version    |
-|-------------------------|-------------------------------|
-| v3.0.0                  | 5.1.3.x                       |
+| Driver      | HPE Nimble Storage Version | Release Notes    |
+|-------------|----------------------------|------------------|
+| v3.0.0      | 5.1.3.x and above          | [v3.0.0.md](release-notes/v3.0.0.md)|
```
Comment (datamattsson) [inline README.md]: `.md` is not necessary within the brackets, only [v3.0.0]

File: release-notes/v3.0.0.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
+# HPE Volume Driver for Kubernetes FlexVolume Plugin 3.0.0 Release Notes
+
+## HPE Volume Driver for Kubernetes FlexVolume Plugin 3.0.0
+
+| **Version:** |3.0.0|
+|--------------|-----|
+| **Revision:** | Monday September 16, 2019 22:05:17|
+
+The release notes describe the major changes, fixes, and known issues for this release of theNew for HPE Volume Driver for Kubernetes FlexVolume Plugin 3.0.0. They do not include all individual fixes and internal changes.
```
Comment (datamattsson) [inline release-notes/v3.0.0.md]: s/theNew/the new/g

File: release-notes/v3.0.0.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+For technical support, contact HPE Nimble Storage Support at:
+[mailto:support@nimblestorage.com](mailto:support@nimblestorage.com)
+877-3-NIMBLE \(877-364-6253\), option 2.
```
Comment (datamattsson) [inline release-notes/v3.0.0.md]: backslashes

**Review by shivamerla — APPROVED** [review]

File: release-notes/v3.0.0.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+You can manually reach the documentation page by logging onto HPE InfoSight and selecting **Resources** \> **Nimble Documentation**.
+
+**Third-Party Software Notices**
+
+All third-party software notices can found in the Documentation Portal on HPE InfoSight.
+
+Here are the steps to manually access the third-party software notices:
+
+1.  Log in to HPE Infosight \([https://infosight.hpe.com/resources/nimble/docs](https://infosight.hpe.com/resources/nimble/docs)\) .
```
Comment (datamattsson) [inline release-notes/v3.0.0.md]: backslashes

File: release-notes/v3.0.0.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+The documentation for HPE Nimble Storage Volume Plugin for Docker is available on the HPE Nimble Storage Github at [https://github.com/hpe-storage/flexvolume-driver/blob/master/README.md](https://github.com/hpe-storage/flexvolume-driver/blob/master/README.md).
+
+The documentation on the hub provides details for setting up and using the product. It contains the most current information about the HPE Volume Driver for Kubernetes FlexVolume Plugin.
+
+These Release Notes and other HPE Nimble Storage user documentation are available on HPE InfoSight:
+
+[https://infosight.hpe.com/resources/nimble/docs](https://infosight.hpe.com/resources/nimble/docs)
+
+You can manually reach the documentation page by logging onto HPE InfoSight and selecting **Resources** \> **Nimble Documentation**.
```
Comment (datamattsson) [inline release-notes/v3.0.0.md]: backslashes

File: release-notes/v3.0.0.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+
+|ID|Component|Title|Description|Workaround|
+|--|---------|-----|-----------|----------|
+|NLT-1630|nlt.docker.vol|XFS Mount Error| If the event of a failover or failback, there may be scenarios in which the mount fails due to errors such as the following: - rc=32 err=mount: /dev/mapper/mpathfhw: can't read superblock - rc=888 mount timed out | Repair the filesystem on the volume: 1. Stop the nimbled service on host \(systemctl stop nimbled\) 2. On the array, add the ACL to the volume. 3. Scan the device from the host and find its mpath name: echo "- - 99" \> /sys/class/scsi\_host/host7/scan echo "- - 99" \> /sys/class/scsi\_host/host8/scan 4. Create a temporary directory to mount the volume: mkdir -p /tmp/kk 5. Mount the volume: mount /dev/mapper/mpathfhw /tmp/kk 6. Repair the filesystem on the volume and then verify the data is recovered: xfs\_repair /dev/mapper/mpathfhw 7. Unmount the volume: umount /tmp/kk 8. On the array, remove the ACL from the volume. 9. Start nimbled service \(systemctl start nimbled\) 10. Once nimbled is up and running, the volume should get mounted from the host and the pod should be able to come up |
```
Comment (datamattsson) [inline release-notes/v3.0.0.md]: This blurb is unreadable

File: release-notes/v3.0.0.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+|NLT-1623|nlt.docker.vol|Ubuntu FC Hosts Mount Failures| With latest multipath packages in Ubuntu 16 and 18, lsscsi and multipath are not in sync and multipath does not contain updated paths, which may lead to mount failures. 1 \# lsscsi | grep :15 \[7:0:4:15\] disk Nimble Server 1.0 /dev/sdc 2 \# cat /sys/block/sdc/device/state running 3 \# sg\_vpd -p di /dev/sdc | grep vendor vendor specific: fc62cac62d184df46c9ce900a5007a7f 4 \# multipath -ll | grep fc62cac62d184df46c9ce900a5007a7f 5 \# multipathd show paths | grep fc62cac62d184df46c9ce900a5007a7f HPE Nimble Storage will be working with Ubuntu to raise this issue with multipath | 1. Execute multipathd reconfigure and multipath commands. 2 . Verify that all the above paths from lsscsi show up in multipath show paths. If not, then contact HPE Nimble Support. After running multipathd reconfigure: \# multipathd show paths | grep :15 7:0:4:15 sdc 8:32 1 active ghost running XXX....... 6/20 \# multipathd show paths format "%w %d %t %i %o %T %c %r %R %m" | grep fc62cac62d184df46c9ce900a5007a7f 2fc62cac62d184df46c9ce900a5007a7f sdc active 7:0:4:15 running ghost tur 0x56c9ce90c75b4704 0x10000090fa8eeff8 mpathog |
```
Comment (datamattsson) [inline release-notes/v3.0.0.md]: Same as above

**Review by datamattsson — CHANGES_REQUESTED** [review]
The workaround columns are a bit unreadable. But I guess this is a text conversion issue.

**Pull request merged**